integreat_cms/static/src/js/tinymce-plugins/custom_link_input/plugin.js

```diff
@@ -85,6 +87,16 @@ import { getCsrfToken } from "../../utils/csrf-token";
             const updateDialog = (api) => {
                 let data = api.getData();
 
+                if (data.autoupdate) {
+                    api.disable("text");
+                    api.setData({
+                        url: "",
+                        text: "",
+                    });
```

Comment:
I think this does not work as is right now: For example, when a user opens the link dialog and enters "Willko" in the search bar, then the url and text will only be visible when the page "Willkommen in augsburg" has been found initially. Subsequent keystrokes however will clear those fields again, making it impossible to click on submit.

Unfortunately I

Reply:
Good catch. I'm getting the urge to refactor the `updateDialog()` function, but that assumes that the logic indeed can be reorganized in a better fashion :sweat_smile: